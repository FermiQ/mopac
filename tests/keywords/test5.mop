gradients  UHF OPEN(2,3) CHARGE=1 ms=-0.5 precise
comment1
comment2
C     0.000000     0.000000     0.000000
H     0.000000     0.000000     1.089000
H     1.026719     0.000000    -0.363000
H    -0.513360    -0.889165    -0.363000
H    -0.513360     0.889165    -0.363000

gradients  OPEN(5,3) CHARGE=1  precise
comment1
comment2
C     0.000000     0.000000     0.000000
H     0.000000     0.000000     1.089000
H     1.026719     0.000000    -0.363000
H    -0.513360    -0.889165    -0.363000
H    -0.513360     0.889165    -0.363000

gradients  OPEN(5,3) CHARGE=1  precise symmetry
 methane

  H     0.00000000 +0    0.0000000 +0    0.0000000 +0     0     0     0
  C     1.08720813 +1    0.0000000 +0    0.0000000 +0     1     0     0
  H     1.08720813 +0  109.4712210 +0    0.0000000 +0     2     1     0
  H     1.08720813 +0  109.4712210 +0 -120.0000000 +0     2     1     3
  H     1.08720813 +0  109.4712210 +0  120.0000000 +0     2     1     3
 
2 1 3 4 5
