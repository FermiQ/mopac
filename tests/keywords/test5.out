 *******************************************************************************
 ** Site#:    0         For non-commercial use only           Version 21.207M **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 21.207M, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web-site: HTTP://OpenMOPAC.net.          **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Fri Jul 30 15:19:51 2021  *
 *  UHF        - UNRESTRICTED HARTREE-FOCK CALCULATION
 *
 *
 *
 *                 CHARGE ON SYSTEM = +1
 *
 *
 *
 *  OPEN(M,N)  - THERE ARE 2 ELECTRONS IN 3 LEVELS
 *  MS=        - IN MECI, MAGNETIC COMPONENT OF SPIN =-0.5
 *  PRECISE    - TIGHTER CRITERIA TO BE USED
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *  GRADIENTS  - ALL GRADIENTS TO BE PRINTED
 *******************************************************************************
gradients  UHF OPEN(2,3) CHARGE=1 ms=-0.5 precise
 comment1
 comment2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          0.00000000  *   0.00000000  *   0.00000000  *
     2       H          0.00000000  *   0.00000000  *   1.08900000  *
     3       H          1.02671900  *   0.00000000  *  -0.36300000  *
     4       H         -0.51336000  *  -0.88916500  *  -0.36300000  *
     5       H         -0.51336000  *   0.88916500  *  -0.36300000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         C          0.0000      0.0000      0.0000
     2         H          0.0000      0.0000      1.0890
     3         H          1.0267      0.0000     -0.3630
     4         H         -0.5134     -0.8892     -0.3630
     5         H         -0.5134      0.8892     -0.3630

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: C H4  =     5 atoms



      MOLECULAR POINT GROUP   :   Td  


          UHF CALCULATION, NO. OF ALPHA ELECTRONS =    3
                           NO. OF BETA  ELECTRONS =    4

          Geometry optimization using EF

          DIAGONAL MATRIX USED AS START HESSIAN

 CYCLE:     1 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    85.745 HEAT:  364.3994
 CYCLE:     2 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    38.194 HEAT:  359.7164
 CYCLE:     3 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    14.287 HEAT:  358.7295
 CYCLE:     4 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    12.561 HEAT:  358.4688
 CYCLE:     5 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     4.630 HEAT:  358.3539
 CYCLE:     6 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     2.349 HEAT:  358.3275
 CYCLE:     7 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     1.636 HEAT:  358.3168
 CYCLE:     8 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     1.207 HEAT:  358.3116
 CYCLE:     9 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     0.598 HEAT:  358.3082
 CYCLE:    10 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.187 HEAT:  358.3074
 CYCLE:    11 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     0.084 HEAT:  358.3073
 CYCLE:    12 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     0.063 HEAT:  358.3073
 CYCLE:    13 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.038 HEAT:  358.3073

     GRADIENT =  0.03784 IS LESS THAN CUTOFF =  0.05000



 -------------------------------------------------------------------------------
 gradients  UHF OPEN(2,3) CHARGE=1 ms=-0.5 precise
 comment1
 comment2


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 21.207M)
                                                       Fri Jul 30 15:19:51 2021




          FINAL HEAT OF FORMATION =        358.30732 KCAL/MOL =    1499.15784 KJ/MOL


          TOTAL ENERGY            =       -160.88092 EV
          ELECTRONIC ENERGY       =       -363.08669 EV  POINT GROUP:     Td  
          CORE-CORE REPULSION     =        202.20577 EV
          COSMO AREA              =         55.64 SQUARE ANGSTROMS
          COSMO VOLUME            =         37.84 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.03784 = 0.01692 PER ATOM
          IONIZATION POTENTIAL    =         22.015184 EV
          ALPHA SOMO LUMO (EV)    =        -31.807-18.466
          BETA  SOMO LUMO (EV)    =        -22.015 -3.133
          NO. OF ALPHA ELECTRONS  =          3
          NO. OF BETA  ELECTRONS  =          4
          MOLECULAR WEIGHT        =         16.0426

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     5    H     2     1.89821
            H     4    H     3     1.89803
            H     3    H     2     1.34221


          SCF CALCULATIONS        =         15
 
          WALL-CLOCK TIME         =          0.070 SECONDS
          COMPUTATION TIME        =          0.065 SECONDS



       FINAL  POINT  AND  DERIVATIVES

   PARAMETER     ATOM    TYPE            VALUE        GRADIENT
      1          1  C    CARTESIAN X     0.000001      0.008540  KCAL/ANGSTROM
      2          1  C    CARTESIAN Y     0.000000     -0.000000  KCAL/ANGSTROM
      3          1  C    CARTESIAN Z     0.000046      0.025934  KCAL/ANGSTROM
      4          2  H    CARTESIAN X     0.000308      0.001319  KCAL/ANGSTROM
      5          2  H    CARTESIAN Y    -0.000000     -0.000000  KCAL/ANGSTROM
      6          2  H    CARTESIAN Z     1.162416      0.006881  KCAL/ANGSTROM
      7          3  H    CARTESIAN X     1.095704     -0.013494  KCAL/ANGSTROM
      8          3  H    CARTESIAN Y    -0.000000     -0.000000  KCAL/ANGSTROM
      9          3  H    CARTESIAN Z    -0.387834     -0.007681  KCAL/ANGSTROM
     10          4  H    CARTESIAN X    -0.548008      0.001817  KCAL/ANGSTROM
     11          4  H    CARTESIAN Y    -0.949064     -0.006063  KCAL/ANGSTROM
     12          4  H    CARTESIAN Z    -0.387371     -0.012567  KCAL/ANGSTROM
     13          5  H    CARTESIAN X    -0.548008      0.001817  KCAL/ANGSTROM
     14          5  H    CARTESIAN Y     0.949064      0.006063  KCAL/ANGSTROM
     15          5  H    CARTESIAN Z    -0.387371     -0.012567  KCAL/ANGSTROM




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          0.00000077  *   0.00000000  *   0.00004613  *
     2       H          0.00030762  *  -0.00000000  *   1.16241598  *
     3       H          1.09570384  *  -0.00000000  *  -0.38783398  *
     4       H         -0.54800756  *  -0.94906413  *  -0.38737061  *
     5       H         -0.54800756  *   0.94906413  *  -0.38737061  *

                             CARTESIAN COORDINATES

   1    C        0.000000768     0.000000000     0.000046125
   2    H        0.000307624    -0.000000000     1.162415978
   3    H        1.095703844    -0.000000000    -0.387833984
   4    H       -0.548007558    -0.949064126    -0.387370611
   5    H       -0.548007558     0.949064127    -0.387370611


           Empirical Formula: C H4  =     5 atoms



      MOLECULAR POINT GROUP   :   Td  


          ALPHA EIGENVALUES  
 -31.80658 -18.46617 -18.46553 -18.46536  -2.73623  -2.73619  -2.73582  -2.32135


           BETA EIGENVALUES 
 -32.78611 -22.01602 -22.01535 -22.01518  -3.13320  -3.13315  -3.13274  -2.51572


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          C          -0.239973        4.2400     1.17531     3.06466
    2          H           0.310008        0.6900     0.68999
    3          H           0.309979        0.6900     0.69002
    4          H           0.309993        0.6900     0.69001
    5          H           0.309993        0.6900     0.69001
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.    -0.000     0.000    -0.000     0.000
 HYBRID         0.000    -0.000    -0.000     0.000
 SUM           -0.000     0.000    -0.000     0.000


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  C   1.17531   1.02157   1.02156   1.02153
    2  H   0.68999
    3  H   0.69002
    4  H   0.69001
    5  H   0.69001


                    (SZ)    =   -0.500000
           Average over configurations used, so (S**2) is not meaningful


          ATOMIC ORBITAL SPIN POPULATIONS

    Atom   Total    s     px     py     pz   
         Atomic
    1  C -0.6090  0.009 -0.206 -0.206 -0.206
    2  H -0.0978 -0.098
    3  H -0.0977 -0.098
    4  H -0.0978 -0.098
    5  H -0.0978 -0.098
 *******************************************************************************
 ** Site#:    0         For non-commercial use only           Version 21.207M **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 21.207M, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web-site: HTTP://OpenMOPAC.net.          **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Fri Jul 30 15:19:51 2021  *
 *
 *
 *
 *                 CHARGE ON SYSTEM = +1
 *
 *
 *
 *  OPEN(M,N)  - THERE ARE 5 ELECTRONS IN 3 LEVELS
 *  PRECISE    - TIGHTER CRITERIA TO BE USED
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *  GRADIENTS  - ALL GRADIENTS TO BE PRINTED
 *******************************************************************************
gradients  OPEN(5,3) CHARGE=1  precise
 comment1
 comment2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          0.00000000  *   0.00000000  *   0.00000000  *
     2       H          0.00000000  *   0.00000000  *   1.08900000  *
     3       H          1.02671900  *   0.00000000  *  -0.36300000  *
     4       H         -0.51336000  *  -0.88916500  *  -0.36300000  *
     5       H         -0.51336000  *   0.88916500  *  -0.36300000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         C          0.0000      0.0000      0.0000
     2         H          0.0000      0.0000      1.0890
     3         H          1.0267      0.0000     -0.3630
     4         H         -0.5134     -0.8892     -0.3630
     5         H         -0.5134      0.8892     -0.3630

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: C H4  =     5 atoms



      MOLECULAR POINT GROUP   :   Td  

      THERE ARE    1 DOUBLY FILLED LEVELS


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 1


                       NO. OF LEVELS WITH OCCUPANCY 1.667  =  3

          Geometry optimization using EF

          DIAGONAL MATRIX USED AS START HESSIAN

 CYCLE:     1 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:    76.525 HEAT:  298.7828
 CYCLE:     2 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:   119.874 HEAT:  293.5790
 CYCLE:     3 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:    84.946 HEAT:  288.3668
 CYCLE:     4 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    39.253 HEAT:  279.7787
 CYCLE:     5 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:    33.472 HEAT:  276.2372
 CYCLE:     6 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    16.616 HEAT:  275.4567
 CYCLE:     7 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:    17.217 HEAT:  275.0393
 CYCLE:     8 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    98.993 HEAT:  273.4448
 CYCLE:     9 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    52.567 HEAT:  270.4648
 CYCLE:    10 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:    61.755 HEAT:  266.9516
 CYCLE:    11 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:    66.591 HEAT:  260.1681
 CYCLE:    12 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    31.744 HEAT:  256.3045
 CYCLE:    13 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:    28.415 HEAT:  254.8349
 CYCLE:    14 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    14.341 HEAT:  254.3747
 CYCLE:    15 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    11.289 HEAT:  254.0334
 CYCLE:    16 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     5.936 HEAT:  253.8944
 CYCLE:    17 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     5.219 HEAT:  253.6951
 CYCLE:    18 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     6.274 HEAT:  253.5285
 CYCLE:    19 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     5.927 HEAT:  253.4464
 CYCLE:    20 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     2.124 HEAT:  253.4284
 CYCLE:    21 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     1.169 HEAT:  253.4188
 CYCLE:    22 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.941 HEAT:  253.4144
 CYCLE:    23 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     0.616 HEAT:  253.4138
 CYCLE:    24 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.281 HEAT:  253.4130
 CYCLE:    25 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     0.072 HEAT:  253.4128
 CYCLE:    26 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     0.036 HEAT:  253.4129

     GRADIENT =  0.03621 IS LESS THAN CUTOFF =  0.05000



 -------------------------------------------------------------------------------
 gradients  OPEN(5,3) CHARGE=1  precise
 comment1
 comment2


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 21.207M)
                                                       Fri Jul 30 15:19:51 2021




          FINAL HEAT OF FORMATION =        253.41276 KCAL/MOL =    1060.27899 KJ/MOL


          TOTAL ENERGY            =       -165.42957 EV  STATE:   1 DOUBLET B2  
          ELECTRONIC ENERGY       =       -372.80698 EV  POINT GROUP:     D2d 
          CORE-CORE REPULSION     =        207.37741 EV
          COSMO AREA              =         55.71 SQUARE ANGSTROMS
          COSMO VOLUME            =         37.78 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.03621 = 0.01619 PER ATOM
          NO. OF FILLED LEVELS    =          1
          AND NO. OF OPEN LEVELS  =          3
          MOLECULAR WEIGHT        =         16.0426

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     3    H     2     2.22747
            H     5    H     4     2.22744
            H     4    H     2     0.42006


          SCF CALCULATIONS        =         28
 
          WALL-CLOCK TIME         =          0.250 SECONDS
          COMPUTATION TIME        =          0.245 SECONDS



       FINAL  POINT  AND  DERIVATIVES

   PARAMETER     ATOM    TYPE            VALUE        GRADIENT
      1          1  C    CARTESIAN X     0.000034      0.018170  KCAL/ANGSTROM
      2          1  C    CARTESIAN Y     0.000000      0.000012  KCAL/ANGSTROM
      3          1  C    CARTESIAN Z    -0.000058     -0.025054  KCAL/ANGSTROM
      4          2  H    CARTESIAN X    -0.620083     -0.007333  KCAL/ANGSTROM
      5          2  H    CARTESIAN Y    -0.000000     -0.000002  KCAL/ANGSTROM
      6          2  H    CARTESIAN Z     0.948668      0.013212  KCAL/ANGSTROM
      7          3  H    CARTESIAN X     0.923061     -0.005439  KCAL/ANGSTROM
      8          3  H    CARTESIAN Y    -0.000000      0.000001  KCAL/ANGSTROM
      9          3  H    CARTESIAN Z    -0.657671      0.000948  KCAL/ANGSTROM
     10          4  H    CARTESIAN X    -0.151548     -0.002706  KCAL/ANGSTROM
     11          4  H    CARTESIAN Y    -1.113721      0.003190  KCAL/ANGSTROM
     12          4  H    CARTESIAN Z    -0.145399      0.005441  KCAL/ANGSTROM
     13          5  H    CARTESIAN X    -0.151548     -0.002703  KCAL/ANGSTROM
     14          5  H    CARTESIAN Y     1.113721     -0.003201  KCAL/ANGSTROM
     15          5  H    CARTESIAN Z    -0.145399      0.005441  KCAL/ANGSTROM




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          0.00003369  *   0.00000008  *  -0.00005776  *
     2       H         -0.62008287  *  -0.00000019  *   0.94866781  *
     3       H          0.92306060  *  -0.00000015  *  -0.65767080  *
     4       H         -0.15154752  *  -1.11372081  *  -0.14539878  *
     5       H         -0.15154753  *   1.11372087  *  -0.14539884  *

                             CARTESIAN COORDINATES

   1    C        0.000033694     0.000000079    -0.000057760
   2    H       -0.620082866    -0.000000187     0.948667810
   3    H        0.923060602    -0.000000149    -0.657670800
   4    H       -0.151547515    -1.113720811    -0.145398776
   5    H       -0.151547532     1.113720874    -0.145398841


           Empirical Formula: C H4  =     5 atoms



      MOLECULAR POINT GROUP   :   D2d 


                  EIGENVALUES  
 -32.73519 -21.85937 -21.85916 -16.66961  -4.95658  -2.40980  -2.34331  -2.34311


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          C          -0.310131        4.3101     1.10507     3.20506
    2          H           0.327573        0.6724     0.67243
    3          H           0.327497        0.6725     0.67250
    4          H           0.327531        0.6725     0.67247
    5          H           0.327530        0.6725     0.67247
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.    -0.001    -0.000     0.001     0.001
 HYBRID        -0.000    -0.000     0.000     0.000
 SUM           -0.001    -0.000     0.001     0.001


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  C   1.10507   1.00730   1.17737   1.02038
    2  H   0.67243
    3  H   0.67250
    4  H   0.67247
    5  H   0.67247
 *******************************************************************************
 ** Site#:    0         For non-commercial use only           Version 21.207M **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 21.207M, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web-site: HTTP://OpenMOPAC.net.          **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Fri Jul 30 15:19:51 2021  *
 *  SYMMETRY   - SYMMETRY CONDITIONS TO BE IMPOSED
 *
 *
 *
 *                 CHARGE ON SYSTEM = +1
 *
 *
 *
 *  OPEN(M,N)  - THERE ARE 5 ELECTRONS IN 3 LEVELS
 *  PRECISE    - TIGHTER CRITERIA TO BE USED
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *  GRADIENTS  - ALL GRADIENTS TO BE PRINTED
 *******************************************************************************



                    PARAMETER DEPENDENCE DATA

        REFERENCE ATOM      FUNCTION NO.    DEPENDENT ATOM(S)
            2                  1               3    4    5

             DESCRIPTIONS OF THE FUNCTIONS USED
   1      BOND LENGTH    IS SET EQUAL TO THE REFERENCE BOND LENGTH   
 
gradients  OPEN(5,3) CHARGE=1  precise symmetry
 methane

  ATOM    CHEMICAL      BOND LENGTH      BOND ANGLE     TWIST ANGLE 
 NUMBER    SYMBOL       (ANGSTROMS)      (DEGREES)       (DEGREES) 
   (I)                     NA:I           NB:NA:I       NC:NB:NA:I       NA    NB    NC 
     1       H          0.00000000       0.0000000       0.0000000        0     0     0
     2       C          1.08720813  *    0.0000000       0.0000000        1     0     0
     3       H          1.08720813     109.4712210       0.0000000        2     1     0
     4       H          1.08720813     109.4712210    -120.0000000        2     1     3
     5       H          1.08720813     109.4712210     120.0000000        2     1     3
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         H          0.0000      0.0000      0.0000
     2         C          1.0872      0.0000      0.0000
     3         H          1.4496      1.0250      0.0000
     4         H          1.4496     -0.5125     -0.8877
     5         H          1.4496     -0.5125      0.8877

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: C H4  =     5 atoms



      MOLECULAR POINT GROUP   :   Td  

      THERE ARE    1 DOUBLY FILLED LEVELS


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 1


                       NO. OF LEVELS WITH OCCUPANCY 1.667  =  3

          Geometry optimization using BFGS
 ONLY ONE VARIABLE, THEREFORE ENERGY A MINIMUM

 -------------------------------------------------------------------------------
 gradients  OPEN(5,3) CHARGE=1  precise symmetry
 methane



     HEAT OF FORMATION MINIMIZED IN ONE LINE SEARCH           
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 21.207M)
                                                       Fri Jul 30 15:19:52 2021




          FINAL HEAT OF FORMATION =        293.95591 KCAL/MOL =    1229.91151 KJ/MOL


          TOTAL ENERGY            =       -163.67147 EV  STATE:   1 DOUBLET T2  
          ELECTRONIC ENERGY       =       -367.07871 EV  POINT GROUP:     Td  
          CORE-CORE REPULSION     =        203.40724 EV
          COSMO AREA              =         55.70 SQUARE ANGSTROMS
          COSMO VOLUME            =         37.93 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.15472 = 0.06919 PER ATOM
          NO. OF FILLED LEVELS    =          1
          AND NO. OF OPEN LEVELS  =          3
          MOLECULAR WEIGHT        =         16.0426

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     4    H     1     1.88413
            H     5    H     3     1.88413
            H     3    H     4     1.33228


          SCF CALCULATIONS        =          9
 
          WALL-CLOCK TIME         =          0.293 SECONDS
          COMPUTATION TIME        =          0.287 SECONDS



       FINAL  POINT  AND  DERIVATIVES

   PARAMETER     ATOM    TYPE            VALUE        GRADIENT
      1          2  C    BOND            1.153787     -0.154724  KCAL/ANGSTROM




  ATOM    CHEMICAL      BOND LENGTH      BOND ANGLE     TWIST ANGLE 
 NUMBER    SYMBOL       (ANGSTROMS)      (DEGREES)       (DEGREES) 
   (I)                     NA:I           NB:NA:I       NC:NB:NA:I       NA    NB    NC 
     1       H          0.00000000       0.0000000       0.0000000        0     0     0
     2       C          1.15378710  *    0.0000000       0.0000000        1     0     0
     3       H          1.15378710     109.4712210       0.0000000        2     1     0
     4       H          1.15378710     109.4712210    -120.0000000        2     1     3
     5       H          1.15378710     109.4712210     120.0000000        2     1     3

                             CARTESIAN COORDINATES

   1    H        0.000000000     0.000000000     0.000000000
   2    C        1.153787100     0.000000000     0.000000000
   3    H        1.538382808     1.087800908     0.000000000
   4    H        1.538382808    -0.543900454    -0.942063221
   5    H        1.538382808    -0.543900454     0.942063221


           Empirical Formula: C H4  =     5 atoms



      MOLECULAR POINT GROUP   :   Td  


                  EIGENVALUES  
 -32.50617 -20.32554 -20.32554 -20.32554  -2.92693  -2.92693  -2.92693  -2.38505


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          H           0.308707        0.6913     0.69129
    2          C          -0.234827        4.2348     1.16968     3.06515
    3          H           0.308707        0.6913     0.69129
    4          H           0.308707        0.6913     0.69129
    5          H           0.308707        0.6913     0.69129
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000    -0.000     0.000     0.000
 HYBRID         0.000     0.000    -0.000     0.000
 SUM            0.000    -0.000     0.000     0.000


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  H   0.69129
    2  C   1.16968   1.02172   1.02172   1.02172
    3  H   0.69129
    4  H   0.69129
    5  H   0.69129

 **********************
 *                    *
 * JOB ENDED NORMALLY *
 *                    *
 **********************



 TOTAL JOB TIME:             0.61 SECONDS

 == MOPAC DONE ==
